packages/wallet/src/redux/protocols/withdrawing/__tests__/scenarios.ts

```diff
@@ -0,0 +1,158 @@
+import * as states from '../states';
+import * as actions from '../actions';
+import * as transactionActions from '../../transaction-submission/actions';
+import { SharedData } from '../..';
+import * as transactionScenarios from '../../transaction-submission/__tests__/scenarios';
+import {
+  ChannelStatus,
+  RUNNING,
+  WAIT_FOR_UPDATE,
+  ChannelState,
+} from '../../../channel-state/state';
+import * as testScenarios from '../../../__tests__/test-scenarios';
+import { EMPTY_OUTBOX_STATE } from '../../../outbox/state';
+import { Wallet } from 'ethers';
+
+// To test all paths through the state machine we will use 4 different scenarios:
+//
+// 1. Happy path: WaitForApproval -> WaitForTransaction -> WaitForAcknowledgement -> Success
+// 2. Withdrawal Rejected: WaitForApproval -> Failure
+// 3. Transaction failure: WaitForApproval -> WaitForTransaction -> Failure
+// 4. Channel not closed failure: WitForApproval -> Failure
```

Comment:
I'm beginning to think that the readme might be a good place to outline the scenarios?